test/cpp/common/timer_test.cc

```diff
@@ -157,8 +157,7 @@ TEST_F(TimerTest, CancelSomeTimers) {
 
 // Enable the following test after
 // https://github.com/grpc/grpc/issues/20049 has been fixed.
-#if 0
-TEST_F(TimerTest, TimerNotCanceled) {
+TEST_F(TimerTest, DISABLED_TimerNotCanceled) {
```

Comment:
`DISABLED_` prefix is preferred instead of commenting out disabled tests as it ensures that the test is compiled (and thus won't rot) while it's disabled. More details in https://github.com/google/googletest/blob/master/googletest/docs/advanced.md#temporarily-disabling-tests.